src/diagnostic/index.ts

```diff
@@ -28,6 +28,7 @@ export async function updateDiagnostic(document: vscode.TextDocument) {
     if (!configuration.depsVersionCheck.enable) return;
 
     if (
+        document.uri.fsPath.includes('node_modules') ||
```

Comment:
this would be better to controlled by a configuration.
It would be helpful when debug some deps tree issues.